icu4j/main/classes/core/src/com/ibm/icu/text/CurrencyDisplayNames.java

```diff
@@ -110,9 +110,10 @@ public static boolean hasData() {
     public abstract ULocale getULocale();
 
     /**
-     * Returns the symbol for the currency with the provided ISO code.  If
-     * there is no data for the ISO code, substitutes isoCode, or returns null
-     * if noSubstitute was set in the factory method.
+     * Returns the symbol for the currency with the provided ISO code.
+     *
+     * If there is no data for this symbol, substitutes isoCode,
```

Comment:
In javadoc (not C++ doxygen), start a new paragraph with `<p>`. No need to terminate it.

(I like to put it inline: ` * <p>If there is no...` but it's also ok to put it into the empty line.)